index.html

```diff
@@ -0,0 +1,138 @@
+<html>
+
+<head>
+    <meta charset="UTF-8">
+    <title>自己紹介</title>
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" media="screen and (min-width: 961px)" type="text/css" href="css/style.css">
+    <link rel="stylesheet" media="screen and (max-width: 960px)" type="text/css" href="css/mobile.css">
+    <link href="https://use.fontawesome.com/releases/v5.6.1/css/all.css" rel="stylesheet">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+    <link href="https://fonts.googleapis.com/css2?family=Dancing+Script:wght@700&display=swap" rel="stylesheet">
+    <script src="https://ajax.googleapis.com/ajax/libs/jquery/3.5.1/jquery.min.js"></script>
+    <script>
+ if(window.matchMedia("(max-width: 960px)").matches){
+
+
+    $(function(){
+  $(window).on('load scroll',function (){
+    $('.contents-test').each(function(){
+      //ターゲットの位置を取得
+      var target = $(this).offset().top;
+      console.log("target" + target)
+      //スクロール量を取得
+      var scroll = $(window).scrollTop();
+      //ウィンドウの高さを取得
+      var height = $(window).height();
+      //ターゲットまでスクロールするとフェードインする
+      if (scroll > target - height +800){
+        //クラスを付与
+        console.log(scroll);
+        $(this).addClass('active');
+      }
+    });
+  });
+});
+
+}else{ 
+
+
+ $(function(){
+  $(window).on('load scroll',function (){
+    $('.contents-test').each(function(){
+      //ターゲットの位置を取得
+      var target = $(this).offset().top;
+      console.log("target" + target)
+      //スクロール量を取得
+      var scroll = $(window).scrollTop();
+      //ウィンドウの高さを取得
+      var height = $(window).height();
+      //ターゲットまでスクロールするとフェードインする
+      if (scroll > target - height + 1300){
+        //クラスを付与
+        console.log(scroll);
+        $(this).addClass('active');
+      }
+    });
+  });
+});
+}
+         
+        
+         
+    </script>
+</head>
+
+<body>
+
+
+    <div id="loading">
+        <div class="loader"></div>
+    </div>
+    
+
+
+
+    <header class="header-parent">
+        <h1 class="header-name">Introduction</h1>
+        <a href="https://www.instagram.com/rtroutrsm/" class="header-instaicon" target="_blank"><img src="instaicon.png" class="instaicon"></a>
```

Comment:
既にこの問題は最新の状況では解決されていますが、`target="_blank"`を用いる場合は`rel="noopener noreferrer"`を付与してあげる方が、古いブラウザを利用している人への対応としてはBetter
[参照記事](https://wwg.co.jp/blog/3807)

```suggestion
        <a href="https://www.instagram.com/rtroutrsm/" class="header-instaicon" target="_blank" rel="noopener noreferrer"><img src="instaicon.png" class="instaicon"></a>
```